quick-start-with-tidb.md

```diff
@@ -76,15 +76,15 @@ As a distributed system, a basic TiDB test cluster usually consists of 2 TiDB in
 
 4. Start a new session to access TiDB:
 
-    1. Install the MySQL client. If it is already installed, skip this step.
+    1. Use the TiUP client to connect to TiDBs.
 
         {{< copyable "shell-regular" >}}
 
         ```shell
-        yum -y install mysql
+        tiup client
```

Comment:
This client doesn't support reading SQL files on STDIN, so it will be hard to use it in examples where we suggest loading data. We should skip switching it for now, and improve the client first,